app/src/main/java/com/elavon/converge/inject/AppModule.java

```diff
@@ -47,7 +47,7 @@
 
     public AppModule(final Context context) {
         this.context = context;
-        this.config = new ConfigLoader(context.getResources(), Config.Environment.TEST).load();
+        this.config = new ConfigLoader(context.getResources(), Config.Environment.LIVE).load();
```

Comment:
Please make this change only for release variant and leave develop/debug to test

Reply:
Review now.